optuna/study/study.py

```diff
@@ -58,6 +59,15 @@ class _ThreadLocalStudyAttribute(threading.local):
     cached_all_trials: list["FrozenTrial"] | None = None
 
 
+def _get_feasible_trials(trials: Sequence[FrozenTrial]) -> list[FrozenTrial]:
```

Comment:
It seems that this is identical to that of `study/_multi_objective.py`. Can we remove either one?